### ISSUE TITLE
Load-based scaling

### ISSUE THREAD

**Issue by RAnders00** [robotty/dank-twitch-irc#194]
Create new connections when lots of messages need to be sent

**Comment by 5E7EN** [comment]
Is this in order to sent messages faster than the default Twitch enforced interval of 100ms by distributing the messages across multiple connections? Why can't messages just be queued on a single connection?

**Comment by RAnders00** [comment]
Yes, it's a thing larger bots might work if they consistently have to send messages faster than 100ms intervals. The messages could back up faster than they can be sent. This also seems useful for usages like mass-banning bots or similar.

**Comment by 5E7EN** [comment]
This is a fair request, however there's one main issue I've experienced in the past while working with similar configurations - and that is, messages being sent out of order (which I assume is because of clients fluctuating in latency; along with not actually confirming message delivery) - as seen here: https://i.imgur.com/qaNe9JN.gif (supposed 10-width pyramid).

**Comment by TroyDota** [comment]
@5E7EN @RAnders00 Perhaps creating message groups where messages within the group must be processed on a single connection and then create a connection if no connection can support the size of the group?
